src/chat/summarizers/dockerContainers.ts

```diff
@@ -32,6 +34,29 @@ export function summarizeLocalDockerContainer(container: ContainerInspectRespons
   }
 
   summary.appendMarkdown(`\n- Status: ${container.State?.Status}`);
+  return summary.value;
+}
+
+export function summarizeLocalDockerContainer(container: ContainerInspectResponse): string {
+  const containerName = container.Name?.replace("/", "") ?? "";
+  const summary = new MarkdownString().appendMarkdown(`### "${containerName}"`);
+
+  const config = workspace.getConfiguration();
+  const kafkaImageRepo: string = config.get(LOCAL_KAFKA_IMAGE) ?? DEFAULT_KAFKA_IMAGE_REPO;
+  const schemaRegistryImageRepo: string =
+    config.get(LOCAL_SCHEMA_REGISTRY_IMAGE) ?? DEFAULT_SCHEMA_REGISTRY_REPO;
+
+  const image: string | undefined = container.Config?.Image;
+
+  if (!image) {
+    summary.appendMarkdown(`\n- Error: No image configuration found`);
+    return summary.value;
+  }
+
+  if (image.includes(kafkaImageRepo) || image.includes(schemaRegistryImageRepo)) {
+    return summarizeContainer(summary, container, image);
+  }
```

Comment:
It may be easier adding the image repo+tag to `summary` in here rather than passing it to other helper functions